.github/workflows/externalworkloads.yaml

```diff
@@ -85,21 +104,42 @@ jobs:
         run: |
           gcloud compute ssh ${{ env.vmName }} --zone ${{ env.zone }} --command "ping -c 3 \$(cilium service list get -o jsonpath='{[?(@.spec.flags.name==\"clustermesh-apiserver\")].spec.backend-addresses[0].ip}')"
 
-      - name: Connectivity test
+      - name: Load cilium test script in configmap
         run: |
-          cilium connectivity test --all-flows
+          kubectl create configmap cilium-cli-test-script -n kube-system --from-file=in-cluster-test-script.sh=.github/in-cluster-test-scripts/external-workloads.sh
 
-      - name: Cleanup
+      - name: Create cilium-cli test job
+        run: |
+          helm install .github/cilium-cli-test-job-chart --generate-name \
+          --set tag=${{ steps.vars.outputs.sha }} \
+          --set job_name=cilium-cli \
+          --set test_script_cm=cilium-cli-test-script \
+
+      - name: Wait for test job
+        run: |
+          kubectl -n kube-system wait job/cilium-cli --for=condition=complete --timeout=10m
+
+      - name: Post-test installation logs
+        if: ${{ always() }}
+        run: |
+          kubectl logs -n kube-system job/cilium-cli-install
+
+      - name: Post-test information gathering
         if: ${{ always() }}
         run: |
-          cilium status
-          cilium clustermesh status
-          cilium clustermesh vm status
+          kubectl logs -n kube-system job/cilium-cli
+          gcloud compute ssh ${{ env.vmName }} --zone ${{ env.zone }} --command "cilium status"
```

Comment:
Can we do something similar on other workflows to retrieve `cilium status` as we did before?
Likewise in this workflow for `cilium clustermesh vm status` which seems to have disappeared here.

Reply:
`cilium status` and `cilium clustermesh vm status` is added at the end of test jobs (after connectivity check).